libs/designsystem/range/src/range.component.ts

```diff
@@ -99,6 +99,10 @@ export class RangeComponent implements OnChanges, OnInit, ControlValueAccessor {
     }
   }
 
+  private defaultPinFormatter(value: number): string | number {
+    return value;
+  }
```

Comment:
Don't know if it's because my Typescript skill are rusty 😄, but this looks a bit odd to me.

How can `defaultPinFormatter()` return a string if it returns `value` and `value` is always a number?

Reply:
Fixed! I originally just matched the type signature of the pinFormatter but you are right it is not needed for the `defaultPinFormatter()` function